docs/native-code.md

```diff
@@ -0,0 +1,122 @@
+
+
+
+## Win32 Desktop apps vs. RNW apps
+
+Whether you are new to Windows development, or you are a Win32 desktop app veteran, the following FAQs should answer some common questions.
+
+When you add Windows support to a react native app via the steps described in the [Install the windows extension](getting-started.md#install-the-windows-extension) section, the react-native-windows CLI will create a UWP app for you. 
+
+Therefore, <u>**RNW apps are UWP apps**</u> and therefore have the following characteristics:
+
+### API surface
+The set of APIs these app can access are a subset of all Windows APIs (i.e. those accessible via WinRT). See:
+- [Win32 and COM APIs for UWP apps](https://docs.microsoft.com/en-us/uwp/win32-and-com/win32-and-com-for-uwp-apps)
+- [CRT functions not supported in Universal Windows Platform apps](https://docs.microsoft.com/en-us/cpp/cppcx/crt-functions-not-supported-in-universal-windows-platform-apps)
+- [Alternatives to Windows APIs in Universal Windows Platform (UWP) apps](https://docs.microsoft.com/en-us/uwp/win32-and-com/alternatives-to-windows-apis-uwp)
+
+### Isolation
+The app runs inside of an app container - a type of sandbox. This provides apps with a secure way to install, access system resources like the filesystem, and lets the system manage their lifetime (e.g. suspending the app when it isn't on the foreground). This means that by default an RNW app cannot access arbitrary filesystem locations, start arbitrary processes, etc. UWP apps that need to access these kinds of capabilities may be able to do so via [App capability declarations](https://docs.microsoft.com/en-us/windows/uwp/packaging/app-capability-declarations).
+
+### Packaging
+
+They are signed and packaged. [Packaging](https://docs.microsoft.com/en-us/windows/uwp/packaging/) is a mechanism through which an app and its dependencies acquire an identity, which is used to determine whether API calls that require system capabilities (e.g. filesystem access) should succeed or not.
+
+### Distribution
+They can be distributed, installed and updated in the following ways:
+  - via [the Microsoft Store](https://docs.microsoft.com/en-us/windows/apps/desktop/modernize/desktop-to-uwp-distribute).
+  - via [your private Store](https://docs.microsoft.com/en-us/microsoft-store/distribute-apps-to-your-employees-microsoft-store-for-business) if you are a business or educational organization. See also [Distribute LOB apps to enterprises](https://docs.microsoft.com/en-us/windows/uwp/publish/distribute-lob-apps-to-enterprises).
+  - using [App Installer](https://docs.microsoft.com/en-us/windows/msix/app-installer/installing-windows10-apps-web).
+  - It's worth noting that you cannot just "_copy an exe_" as the app package contains more than just the main executable, including an app manifest, assets, dependent framework libraries, etc.

```

Comment:
This bullet and later bullets don't follow the same pattern as the first three; should this be a separate list? It seems like these bullets are more FYI whereas the first bullets are about how the apps can be distributed.